src/Stratis.Bitcoin.Features.BlockStore/StoreSettings.cs

```diff
@@ -23,6 +23,16 @@ public class StoreSettings
         /// <summary>The maximum size of bytes the cache can contain.</summary>
         public int MaxCacheSize { get; set; }
 
+        /// <summary><c>true</c> to maintain a full transaction index.</summary>
+        public bool TxIndex { get; set; }
```

Comment:
What it was not in store settings before? hmm...

Reply:
It was, then moved to node settings, then back to BS settings again because you've convinced me to put indexer in BS feature